liaisons/opcf/tech_reqs.md

```diff
@@ -0,0 +1,45 @@
+# Technical Requirements for OPC UA/WoT Collaboration
+
+## Use Cases
+See https://w3c.github.io/wot-usecases/#use-case-opc-ua-binding
+
+## Objectives
+1. Use OPC UA servers (device, node) from WoT Consumers; treat an OPC UA server as a Thing
+2. Express metadata for OPC UA servers using WoT TDs
```

Comment:
we may also discuss "what" should be represented in the TD. It is not unusual for a UA server to have tens of thousands of nodes.

Reply:
An OPC UA server of a single device will have the mandatory _Server_ object and then depending on used information model the instance of the device is represents. But as @sebastiankb says, could be a few 10 ... x000, therefore the standardized information models should be taken into account here.

As chair of the OPC UA for Machine Tools, JWG, this is an example https://github.com/OPCFoundation/UA-Nodeset/blob/latest/MachineTool/Machinetool-Example.csv (119 Nodes for a simple machine tool + Server Object).

Reply:
I have seen this same pattern in Home IoT.  In Home Assistant, it is more common to focus on "entities" (single measurements or actuators) rather than "Devices".  Even in my test setup I have some devices with dozens of entities, and when I write automation rules, etc. I focus more on entities than devices.  I suspect OPC UA nodes are more like entities than devices...  

So: should Things map to devices, or entities?  Should OPC UA nodes be Things, or affordances?  If we use Things, can we use collections, etc. to represent the co-location of nodes in a particular device?